source/docs/software/pathplanning/index.rst

```diff
@@ -0,0 +1,11 @@
+Path Planning
+=============
+
+Path Planning is the process of developing and "planning" paths that your robot can autonomously follow. These paths use the WPILib Trajectory Framework for generation and Ramsete for following. This section highlights the process of characterizing your robot for system identification, developing with the trajectory tracking API and visualizing your paths with PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for non-commandbased usage.
```

Comment:
I would remove the non-commandbased mention and mention for more detail.  People will generally go to the generic stuff to dive into more detail.

Reply:
@jasondaming Eh, the Command-Based tutorial is very not useful for non-command teams for 3/4ths of it

Reply:
okay then maybe mention both.  As is it sounds like the only reason people should go to the generic stuff is non-command based and that isn't true